app/services/twilio_verification_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'twilio-ruby'
+
+class TwilioVerificationService
+  CLIENT = Twilio::REST::Client.new(
+    Rails.application.secrets.twilio_account_sid,
+    Rails.application.secrets.twilio_api_key
+  )
+  VERIFY_SERVICE_ID = 'VAa06a66dad4c1ca3c199a46334ff11945'
```

Comment:
this isn't private so it's okay to just hardcode it here

Reply:
Could you add a comment above that line that so someone doesn't put it in secrets later?